### DISCUSSION TITLE
Can I increase the number of patients with allergies?

### DISCUSSION THREAD

**Discussion by nikhil-sharma2** [synthetichealth/synthea#1060] [Q&A]
Hi everyone, I am trying to generate data with more patients having allergies. Currently, I am only getting 4-5 patients out of 40 male patients. I need at least 50-60% of patients to have any sort of allergies.

Is there any way to control this? Any kind of help is appreciated, thanks! :)

**Comment by jawalonoski** [comment]
You can control this by modifying the modules.

Follow the procedures described in https://github.com/synthetichealth/synthea/wiki/Generic-Module-Framework:-Running-New-Modules#modifying-existing-modules

The modules you want to edit are probably (this is a subset of all allergy modules, you can see the full list in the Module Builder):

- https://synthetichealth.github.io/module-builder/#allergies
- https://synthetichealth.github.io/module-builder/#allergies/drug_allergy_incidence
- https://synthetichealth.github.io/module-builder/#allergies/environmental_allergy_incidence
- https://synthetichealth.github.io/module-builder/#allergies/food_allergy_incidence
- https://synthetichealth.github.io/module-builder/#allergies/outgrow_env_allergies
- https://synthetichealth.github.io/module-builder/#allergies/outgrow_food_allergies

**Reply by nikhil-sharma2** [reply]
@jawalonoski - thanks a lot for the quick reply. I am new to Synthea's codebase so troubling you again with a small query here.
Let's say I want to increase the occurrence of `Bee_Allergy` can you please tell me the change that I need to do in this module? Attaching photo for reference:

![Capture](https://user-images.githubusercontent.com/89768194/167631960-1197ef0e-8a0c-4fb5-a6c3-4127eb758efb.JPG)

**Comment by jawalonoski** [comment]
Click on "Chance_of_Bee_Allergy" and on editing panel (on the left), find the "Weight" fields and adjust the probability as you prefer.

![image](https://user-images.githubusercontent.com/1054765/167632938-ae734eb5-45fa-4316-b677-1fe56c3dbb90.png)

**Reply by nikhil-sharma2** [reply]
@jawalonoski - you are a lifesaver, thanks a ton!! 🚀 😄
